vcluster/deploy/topologies/air-gapped.mdx

```diff
@@ -0,0 +1,377 @@
+
+  **Verify & Use**:
+  ```bash
+  # Verify registry
+  curl http://localhost:5001/v2/_catalog
+  ```
+
+  </details>
+
+### Download vCluster Helm chart
+
+Download the `vCluster` Helm chart from the Loft Helm repository.
+
+:::tip
+To retrieve all available versions of the vCluster Helm chart, run the following command:
+
+```bash title="Show latest 10 vCluster chart versions"
+helm repo update
+helm search repo loft/vcluster --versions | grep "^loft/vcluster" | head
+```
+:::
+
+```bash title="Download vCluster Helm chart"
+export CHART_VERSION="0.21.1"  # Replace with the desired version
+curl -O https://charts.loft.sh/charts/vcluster-"${CHART_VERSION}".tgz
+```
+
+:::info
+The `vCluster` Helm chart is a tarball that contains all the necessary images
+to deploy the vCluster control plane.
+:::
+
+### Download and push required container images
+
+For clusters unable to pull images from Docker Hub, you need to push the
+platform images to your private registry. Each vCluster release includes a
+`vcluster-images.txt` file that lists the necessary images.
+
+:::warning
+When using virtual clusters in air-gapped environments, the
+`config.experimental.deploy.vcluster.helm` [configuration setting](/docs/vcluster/configure/vcluster-yaml/#experimental-deploy-vcluster) does not work with external Helm repositories since they cannot be accessed. This means custom Helm charts from repositories like `charts.bitnami.com` cannot be used for virtual cluster deployments.
+:::
+
+Follow these instructions to download all vCluster images and import them to your private registry:
+
+<Flow id="offline-images">
+<Step>
+
+Set environment variables for the setup process:
+
+      ```bash title="Export environment variables"
+      export VCLUSTER_SERVICE="vcluster"
+      export VCLUSTER_NAMESPACE="vcluster"
+      export REGISTRY=ecr.io/myteam      # This should be a prefix; do not include any LOFT_IMAGE paths
+      ```
+  <Tabs>
+    <TabItem value="existing" label="Existing vCluster Installation" default>
+
+      Retrieve the vCluster version and set the `VERSION` variable:
+      ```bash title="Retrieve and set VERSION"
+      CHART=$(kubectl get service "${VCLUSTER_SERVICE}" -n "${VCLUSTER_NAMESPACE}" -o jsonpath={.metadata.labels.chart})
+      export VERSION=${CHART#vcluster-}   # Remove 'vcluster-' prefix
+      ```
+    </TabItem>
+    <TabItem value="fresh" label="No vCluster Installation">
+      Set the `VERSION` variable to the latest vCluster version you want to install:
+
+
+      ```bash title="Set VERSION variable for fresh installation"
+      export VERSION=<latest-version>  # e.g., "0.21.1"
+      ```
+
+    </TabItem>
+
+  </Tabs>
+</Step>
+  <Step>
+
+Download the `vcluster-images.txt` file and the required scripts
+`download-images.sh` and `push-images.sh`, then make them executable.
+
+:::tip
+Note that the
+`vcluster-images.txt` contains all the distros and versions. You can edit the
+file and remove images you do not need.
+:::
```

Comment:
we should include that there are specific files for version and distro as well:

Example `vcluster-images-k8s-1.30.txt` which is in every release assets: https://github.com/loft-sh/vcluster/releases/tag/v0.21.2